frontend/src/mobile/index.js

```diff
@@ -4,7 +4,7 @@ import Vista from './vista';
 import { reacciones } from './actions';
 
 const Mobile = ({
-  usuario, dispatch, tema, queuedParticipants, ...props
+  usuario, mostrarBotonesParaNavegarEntrePantallas: mostrarBotonesParaNavegarEntrePantallas, dispatch, tema, queuedParticipants, ...props
```

Comment:
si el nombre de la prop y el de la variable son el mismo se puede evitar repetirlos

```suggestion
  usuario, mostrarBotonesParaNavegarEntrePantallas, dispatch, tema, queuedParticipants, ...props
```

Reply:
Corregido!